django_simple_bulma/finders.py

```diff
@@ -40,26 +41,37 @@ def __init__(self):
     def _get_bulma_css(self) -> str:
         """Compiles the bulma css file and returns its relative path."""
         # Start by unpacking the users custom variables
-        scss_string = ""
+        scss_string = '@charset "utf-8";\n'
         for var, value in self.variables.items():
             scss_string += f"${var}: {value};\n"
 
         # SASS wants paths with forward slash:
         sass_bulma_path = str(self.simple_bulma_path).replace('\\', '/')
-        # Now load bulma
-        scss_string += f'@import "{sass_bulma_path}/bulma.sass";'
+        sass_bulma_submodule_path = str(self.bulma_submodule_path).replace('\\', '/')
```

Comment:
I think pathlibs `.as_posix()` method solves this, actually. Could you test if it works if we just do `self.bulma_submodule_path.as_posix()`?

If this works, do that for the other replaces too?

Reply:
There is one replace call further down that cannot be replaced because it is used on a str instead of a Path.